### PR TITLE
19793 - Add tests for Significant Individual Change Date.

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:19793 *https://github.com/bcgov/entity/issues/19793 

*Description of changes:*
Add tests for Significant Individual Change Date. 
this test is to verify that no existing SIs have their significant date changed when we select/add new SI.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Review by kialj876 — COMMENTED** [review]
Looking good - just had a couple small comments

File: btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts [hunk 1]
```diff
@@ -96,4 +96,78 @@ describe('pages -> Beneficial Owner Change', () => {
...
+
+      // enter tax number and select tax residency
+      cy.get('[name="taxNumber"]').type(testData.profile1.taxNumber)
+      cy.get('[data-cy="testTaxResidency"]').get('[type="radio"][value="true"]').check()
+
+      // click 'Done' button to add the individual
+      cy.get('[data-cy=new-si-done-btn]').click()
+    })
+    // verify only new entry has date set for today. All other elements should have different dates.
+    cy.get('td:contains("' + expectedDate + '")').should('have.length', 2)
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts]: As it was searching for td, no, it was supposed to be 2, but I updated it to search for different thing, and to be 1.

File: btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts [hunk 2]
```diff
@@ -96,4 +96,78 @@ describe('pages -> Beneficial Owner Change', () => {
     cy.get('[data-cy=button-control-right-button]').eq(0).click()
     cy.url().should('include', '/review-confirm')
   })
+
+  it('on update Significant Individual Filing Effective Date ' +
+    'verifies that only newly added items have their date changed ', () => {
+    cy.fixture('plotsEntityExistingSiResponse').then((plotsEntityExistingSiResponse) => {
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts:102]: I moved it back as it interferes with some tests that are expecting empty summary table. I will pickup probably next task for refactoring cypress with wait, and have some ideas how to make cypress tests bit nicer :D (been reading a bit on it)

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-126-yx3m12q9.web.app

File: btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (kialj876) [reply btr-web/btr-main-app/cypress/e2e/pages/ownerChange.cy.ts:102]: alright well those ones should have an intercept that just returns the 404 / empty list

**Review by kialj876 — APPROVED** [review]

**Pull request merged**